src/reference/asciidoc/whats-new.adoc

```diff
@@ -48,3 +48,6 @@ See <<streams-config, Streams Configuration>> for more information.
 Now all the JSON-aware components are configured by default with a Jackson `ObjectMapper` produced by the `JacksonUtils.enhancedObjectMapper()`.
 The `JsonDeserializer` now provides `TypeReference`-based constructors for better handling of target generic container types.
 See its JavaDocs and <<serdes>> for more information.
+
+A `ByteArrayJsonMessageConverter` has been as well as a new super class for all Json converters, `JsonMessageConverter`.
```

Comment:
Look like something has missed in this sentence. "Has been" what ?

Reply:
has been **provided**